Revert "Jazmyn week1"

jazmynCortez opened the pull request:
Reverts learn-academy-2020-bravo/week-1-assessment-jazmynCortez#1

jazmynCortez committed:
Revert "Jazmyn week1"

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -10,54 +10,23 @@ var temp1 = 35
-  } else {
-    return "broken"
-  }
-}
```
good job!

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -10,54 +10,23 @@ var temp1 = 35
-//Create for loop
-for(let i=0; i<arr.length; i++){
-  //console.log() and have the array being multiplied by 5
-  console.log(arr[i] * 5)
```
this one is _aaalllmost_ there! yes, we are console logging the correct values but a function requires a return. instead, you'd want to _push_ those values into a new array and then return that array. you're also going t want to wrap all of this logic into a function:

```javascript
const multBy5 = (arr) => {
  let multipliedValues = []
  for(let i=0; i < arr.length; i++) {
    multipliedValues.push(arr[i] * 5)
  }
  return multipliedValues
}
console.log(multBy5(myNumbers1));
```

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -10,54 +10,23 @@ var temp1 = 35
-//create a const variable to make sure the variable name doesn't change
-//make it a function
-//use the map method to be able to go thorugh the array and multiply each index by 5
-const multiplyer = myNumbers2.map(value => value * 5)
```
this one is also suuuuper close! this should just be wrapping a function. right now it is just a variable with a `const` assignment. like this:
```javascript
const multiplyer = (arr) => arr.map(value => value * 5)
```

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -66,68 +35,26 @@ console.log(multiplyer);
-}
-//console.log() both variables given
-console.log(noVowels(stringWithVowels1));
-console.log(noVowels(stringWithVowels2))
```
good job!

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -66,68 +35,26 @@ console.log(multiplyer);
- //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
- let array = string.split("")
- //create a return statement that will filter through the array and add in .join to join the constanants together
- return array.filter(value => vowels.indexOf(value) === -1).join("")
```
this function needs a bit of refactoring to add some logic to check if the input is actually a string

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -66,68 +35,26 @@ console.log(multiplyer);
-//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find "cats"
-var animal = toonimals.filter(value => value.animal === "cat")
-//console.log() the code to call it
-console.log(animal);
```
yay! your variable is checking for all the cats. now we just need to wrap it all in a function to make it reusable.

rachaelstanislaw commented on code-challenges.js:
```diff
@@ -66,68 +35,26 @@ console.log(multiplyer);
-//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find non cat names
-var animal = toonimals.filter(value => value.animal !== "cat")
-//console.log() the code to call it
-console.log(animal);
```
almost! not quite but you're almost there. this solution again, is just a variable, not a function. also, it will return the whole object of each non-cat. we want just the _names_ of the non-cats

rachaelstanislaw commented on interview-questions.md:
```diff
@@ -4,64 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
-  Your answer: tells the computer what to do with variables it's given
+  Your answer:
 
-  Researched answer: A function is a set of instructions detailing how to do a task
```
good research!

rachaelstanislaw commented on interview-questions.md:
```diff
@@ -4,64 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
-  Your answer: filter goes through the index and only outputs the indexes you chose to have at the end exactly. (I'm not sure how to describe map and their differences...)
+  Your answer:
 
-  Researched answer: map function iterates through an array and returns a new array of the same length rather filter function loops through an array and returns a new array with only the values that are truthy or the values that satisfy your stated condition.
```
good research! i hope this cleared it up a bit for you :)

rachaelstanislaw commented on interview-questions.md:
```diff
@@ -4,64 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
 
 3. What is the difference between console.log() and return?
 
-  Your answer: console.log() is used when debugging mostly because information gets inputed to the log that goes into the function to make the function do something, rather a return is part of the function.
```
omg yes!

rachaelstanislaw commented on interview-questions.md:
```diff
@@ -4,64 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
 
 4. In regards to functions, what is an argument?
 
-  Your answer: an argument is what goes inside the () right?....
```
it sure is! it is outside information that the function needs in order to run

rachaelstanislaw commented on interview-questions.md:
```diff
@@ -4,64 +4,64 @@ Answer the following questions. First, without external resources. Challenge you
-  Your answer: TDD is like pseudocode but it is used when using jest. When using TDD it always starts with a describe statement, a test statement, and an expect statement.
+  Your answer:
 
-  Researched answer: Test-driven development is a software development process that relies on the repetition of a very short development cycle: requirements are turned into very specific test cases, then the code is improved so that the tests pass.
```
yippee!

rachaelstanislaw reviewed (approved):
yay! excellent work on your first assessment! i think we are missing the jest file though. let me know if you have questions about any of the feedback. then, you're good to merge your code to master :)

jazmynCortez commented:
Oh no! How do I add that?


On Mon, May 4, 2020 at 4:27 PM Rachael Stanislaw <notifications@github.com>
wrote:

> *@rachaelstanislaw* approved this pull request.
>
> yay! excellent work on your first assessment! i think we are missing the
> jest file though. let me know if you have questions about any of the
> feedback. then, you're good to merge your code to master :)
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419782696>
> :
>
> > -    return `${temp} is below boiling point`
> -  }
> -  //Create an else if statement stating that temp is above boiling point
> -  else if (temp > 212) {
> -    //return "350 is above boiling point"
> -    return `${temp} is above boiling point`
> -  }
> -  //Create an else if statement stating that temp is at boiling point
> -  else if (temp === 212){
> -    //return "212 is at boiling point"
> -    return `${temp} is at boiling point`
> -    //Create else "broken"
> -  } else {
> -    return "broken"
> -  }
> -}
>
> good job!
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419783050>
> :
>
> >
>
>  // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
>  // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
>
> -var arr = [3, 7, 0, 6, -9]
> -//Create for loop
> -for(let i=0; i<arr.length; i++){
> -  //console.log() and have the array being multiplied by 5
> -  console.log(arr[i] * 5)
>
> this one is *aaalllmost* there! yes, we are console logging the correct
> values but a function requires a return. instead, you'd want to *push*
> those values into a new array and then return that array. you're also going
> t want to wrap all of this logic into a function:
>
> const multBy5 = (arr) => {
>   let multipliedValues = []
>   for(let i=0; i < arr.length; i++) {
>     multipliedValues.push(arr[i] * 5)
>   }
>   return multipliedValues
> }console.log(multBy5(myNumbers1));
>
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784425>
> :
>
> >
>  // --------------------3) Create a function that multiplies each number in the array by 5 using map.
>  // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
>
>  var myNumbers2 = [8, -7, 0, 6, 2]
> -//create a const variable to make sure the variable name doesn't change
> -//make it a function
> -//use the map method to be able to go thorugh the array and multiply each index by 5
> -const multiplyer = myNumbers2.map(value => value * 5)
>
> this one is also suuuuper close! this should just be wrapping a function.
> right now it is just a variable with a const assignment. like this:
>
> const multiplyer = (arr) => arr.map(value => value * 5)
>
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784594>
> :
>
> > @@ -66,68 +35,26 @@ console.log(multiplyer);
>  var stringWithVowels1 = "HeyThereLearnStudent"
>  var stringWithVowels2 = "ILoveJavaScript"
>
> -//create const variable connected to the function and put the term string within the parenthesis
> -const noVowels = (string) => {
> -  //create a let variable stating what vowels are
> - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
> - //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
> - let array = string.split("")
> - //create a return statement that will filter through the array and add in .join to join the constanants together
> - return array.filter(value => vowels.indexOf(value) === -1).join("")
> -}
> -//console.log() both variables given
> -console.log(noVowels(stringWithVowels1));
> -console.log(noVowels(stringWithVowels2))
>
> good job!
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784925>
> :
>
> >
> -//create const variable connected to the function and put the term string within the parenthesis
> -const noVowels = (string) => {
> -  //create a let variable stating what vowels are
> - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
> - //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
> - let array = string.split("")
> - //create a return statement that will filter through the array and add in .join to join the constanants together
> - return array.filter(value => vowels.indexOf(value) === -1).join("")
>
> this function needs a bit of refactoring to add some logic to check if the
> input is actually a string
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785083>
> :
>
> >
>  // --------------------6) Create a function that takes the toonimals array and returns an array with only the toon objects that are cats.
>  // Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
>
> -//make all variables stackes above one another
> -var toonimals = [
> -  {name: "Itchy", animal: "mouse"},
> -  {name: "Stimpy", animal: "cat"},
> -  {name: "Daffy", animal: "duck"},
> -  {name: "Scratchy", animal: "cat"},
> -  {name: "Ren", animal: "dog"},
> -  {name: "Felix", animal: "cat"}]
> -//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find "cats"
> -var animal = toonimals.filter(value => value.animal === "cat")
> -//console.log() the code to call it
> -console.log(animal);
>
> yay! your variable is checking for all the cats. now we just need to wrap
> it all in a function to make it reusable.
> ------------------------------
>
> In code-challenges.js
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785520>
> :
>
> >
>  // --------------------7) Using the toonimals variable from #6, create a function that returns only the names of the non-cats.
>  // Expected output: "Itchy" "Daffy" "Ren"
> -//make all variables stackes above one another
> -var toonimals = [
> -  {name: "Itchy", animal: "mouse"},
> -  {name: "Stimpy", animal: "cat"},
> -  {name: "Daffy", animal: "duck"},
> -  {name: "Scratchy", animal: "cat"},
> -  {name: "Ren", animal: "dog"},
> -  {name: "Felix", animal: "cat"}]
> -//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find non cat names
> -var animal = toonimals.filter(value => value.animal !== "cat")
> -//console.log() the code to call it
> -console.log(animal);
>
> almost! not quite but you're almost there. this solution again, is just a
> variable, not a function. also, it will return the whole object of each
> non-cat. we want just the *names* of the non-cats
> ------------------------------
>
> In interview-questions.md
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785591>
> :
>
> >
> -  Researched answer: A function is a set of instructions detailing how to do a task
>
> good research!
> ------------------------------
>
> In interview-questions.md
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785777>
> :
>
> >
> -  Researched answer: map function iterates through an array and returns a new array of the same length rather filter function loops through an array and returns a new array with only the values that are truthy or the values that satisfy your stated condition.
>
> good research! i hope this cleared it up a bit for you :)
> ------------------------------
>
> In interview-questions.md
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785838>
> :
>
> >
>
>
>  3. What is the difference between console.log() and return?
>
> -  Your answer: console.log() is used when debugging mostly because information gets inputed to the log that goes into the function to make the function do something, rather a return is part of the function.
>
> omg yes!
> ------------------------------
>
> In interview-questions.md
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785953>
> :
>
> >
>
>
>  4. In regards to functions, what is an argument?
>
> -  Your answer: an argument is what goes inside the () right?....
>
> it sure is! it is outside information that the function needs in order to
> run
> ------------------------------
>
> In interview-questions.md
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419786075>
> :
>
> >
> -  Researched answer: Test-driven development is a software development process that relies on the repetition of a very short development cycle: requirements are turned into very specific test cases, then the code is improved so that the tests pass.
>
> yippee!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#pullrequestreview-405410218>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APIFD7LQH42H6QOWELN6WHLRP5FNHANCNFSM4MY5BFZQ>
> .
>

jazmynCortez commented:
Cause I see the jest file within git hub but I think your asking for what
was shown in terminal?

On Mon, May 4, 2020 at 4:53 PM Jazmyn Cortez <jazmyncortez@gmail.com> wrote:

> Oh no! How do I add that?
>
>
> On Mon, May 4, 2020 at 4:27 PM Rachael Stanislaw <notifications@github.com>
> wrote:
>
>> *@rachaelstanislaw* approved this pull request.
>>
>> yay! excellent work on your first assessment! i think we are missing the
>> jest file though. let me know if you have questions about any of the
>> feedback. then, you're good to merge your code to master :)
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419782696>
>> :
>>
>> > -    return `${temp} is below boiling point`
>> -  }
>> -  //Create an else if statement stating that temp is above boiling point
>> -  else if (temp > 212) {
>> -    //return "350 is above boiling point"
>> -    return `${temp} is above boiling point`
>> -  }
>> -  //Create an else if statement stating that temp is at boiling point
>> -  else if (temp === 212){
>> -    //return "212 is at boiling point"
>> -    return `${temp} is at boiling point`
>> -    //Create else "broken"
>> -  } else {
>> -    return "broken"
>> -  }
>> -}
>>
>> good job!
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419783050>
>> :
>>
>> >
>>
>>  // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
>>  // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
>>
>> -var arr = [3, 7, 0, 6, -9]
>> -//Create for loop
>> -for(let i=0; i<arr.length; i++){
>> -  //console.log() and have the array being multiplied by 5
>> -  console.log(arr[i] * 5)
>>
>> this one is *aaalllmost* there! yes, we are console logging the correct
>> values but a function requires a return. instead, you'd want to *push*
>> those values into a new array and then return that array. you're also going
>> t want to wrap all of this logic into a function:
>>
>> const multBy5 = (arr) => {
>>   let multipliedValues = []
>>   for(let i=0; i < arr.length; i++) {
>>     multipliedValues.push(arr[i] * 5)
>>   }
>>   return multipliedValues
>> }console.log(multBy5(myNumbers1));
>>
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784425>
>> :
>>
>> >
>>  // --------------------3) Create a function that multiplies each number in the array by 5 using map.
>>  // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
>>
>>  var myNumbers2 = [8, -7, 0, 6, 2]
>> -//create a const variable to make sure the variable name doesn't change
>> -//make it a function
>> -//use the map method to be able to go thorugh the array and multiply each index by 5
>> -const multiplyer = myNumbers2.map(value => value * 5)
>>
>> this one is also suuuuper close! this should just be wrapping a function.
>> right now it is just a variable with a const assignment. like this:
>>
>> const multiplyer = (arr) => arr.map(value => value * 5)
>>
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784594>
>> :
>>
>> > @@ -66,68 +35,26 @@ console.log(multiplyer);
>>  var stringWithVowels1 = "HeyThereLearnStudent"
>>  var stringWithVowels2 = "ILoveJavaScript"
>>
>> -//create const variable connected to the function and put the term string within the parenthesis
>> -const noVowels = (string) => {
>> -  //create a let variable stating what vowels are
>> - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
>> - //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
>> - let array = string.split("")
>> - //create a return statement that will filter through the array and add in .join to join the constanants together
>> - return array.filter(value => vowels.indexOf(value) === -1).join("")
>> -}
>> -//console.log() both variables given
>> -console.log(noVowels(stringWithVowels1));
>> -console.log(noVowels(stringWithVowels2))
>>
>> good job!
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784925>
>> :
>>
>> >
>> -//create const variable connected to the function and put the term string within the parenthesis
>> -const noVowels = (string) => {
>> -  //create a let variable stating what vowels are
>> - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
>> - //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
>> - let array = string.split("")
>> - //create a return statement that will filter through the array and add in .join to join the constanants together
>> - return array.filter(value => vowels.indexOf(value) === -1).join("")
>>
>> this function needs a bit of refactoring to add some logic to check if
>> the input is actually a string
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785083>
>> :
>>
>> >
>>  // --------------------6) Create a function that takes the toonimals array and returns an array with only the toon objects that are cats.
>>  // Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
>>
>> -//make all variables stackes above one another
>> -var toonimals = [
>> -  {name: "Itchy", animal: "mouse"},
>> -  {name: "Stimpy", animal: "cat"},
>> -  {name: "Daffy", animal: "duck"},
>> -  {name: "Scratchy", animal: "cat"},
>> -  {name: "Ren", animal: "dog"},
>> -  {name: "Felix", animal: "cat"}]
>> -//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find "cats"
>> -var animal = toonimals.filter(value => value.animal === "cat")
>> -//console.log() the code to call it
>> -console.log(animal);
>>
>> yay! your variable is checking for all the cats. now we just need to wrap
>> it all in a function to make it reusable.
>> ------------------------------
>>
>> In code-challenges.js
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785520>
>> :
>>
>> >
>>  // --------------------7) Using the toonimals variable from #6, create a function that returns only the names of the non-cats.
>>  // Expected output: "Itchy" "Daffy" "Ren"
>> -//make all variables stackes above one another
>> -var toonimals = [
>> -  {name: "Itchy", animal: "mouse"},
>> -  {name: "Stimpy", animal: "cat"},
>> -  {name: "Daffy", animal: "duck"},
>> -  {name: "Scratchy", animal: "cat"},
>> -  {name: "Ren", animal: "dog"},
>> -  {name: "Felix", animal: "cat"}]
>> -//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find non cat names
>> -var animal = toonimals.filter(value => value.animal !== "cat")
>> -//console.log() the code to call it
>> -console.log(animal);
>>
>> almost! not quite but you're almost there. this solution again, is just a
>> variable, not a function. also, it will return the whole object of each
>> non-cat. we want just the *names* of the non-cats
>> ------------------------------
>>
>> In interview-questions.md
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785591>
>> :
>>
>> >
>> -  Researched answer: A function is a set of instructions detailing how to do a task
>>
>> good research!
>> ------------------------------
>>
>> In interview-questions.md
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785777>
>> :
>>
>> >
>> -  Researched answer: map function iterates through an array and returns a new array of the same length rather filter function loops through an array and returns a new array with only the values that are truthy or the values that satisfy your stated condition.
>>
>> good research! i hope this cleared it up a bit for you :)
>> ------------------------------
>>
>> In interview-questions.md
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785838>
>> :
>>
>> >
>>
>>
>>  3. What is the difference between console.log() and return?
>>
>> -  Your answer: console.log() is used when debugging mostly because information gets inputed to the log that goes into the function to make the function do something, rather a return is part of the function.
>>
>> omg yes!
>> ------------------------------
>>
>> In interview-questions.md
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785953>
>> :
>>
>> >
>>
>>
>>  4. In regards to functions, what is an argument?
>>
>> -  Your answer: an argument is what goes inside the () right?....
>>
>> it sure is! it is outside information that the function needs in order to
>> run
>> ------------------------------
>>
>> In interview-questions.md
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419786075>
>> :
>>
>> >
>> -  Researched answer: Test-driven development is a software development process that relies on the repetition of a very short development cycle: requirements are turned into very specific test cases, then the code is improved so that the tests pass.
>>
>> yippee!
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#pullrequestreview-405410218>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/APIFD7LQH42H6QOWELN6WHLRP5FNHANCNFSM4MY5BFZQ>
>> .
>>
>

rachaelstanislaw commented:
Hmm you should be able to! We can walk through it tomorrow :)

> On May 4, 2020, at 4:54 PM, jazmynCortez <notifications@github.com> wrote:
> 
> 
> Oh no! How do I add that?
> 
> 
> On Mon, May 4, 2020 at 4:27 PM Rachael Stanislaw <notifications@github.com>
> wrote:
> 
> > *@rachaelstanislaw* approved this pull request.
> >
> > yay! excellent work on your first assessment! i think we are missing the
> > jest file though. let me know if you have questions about any of the
> > feedback. then, you're good to merge your code to master :)
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419782696>
> > :
> >
> > > - return `${temp} is below boiling point`
> > - }
> > - //Create an else if statement stating that temp is above boiling point
> > - else if (temp > 212) {
> > - //return "350 is above boiling point"
> > - return `${temp} is above boiling point`
> > - }
> > - //Create an else if statement stating that temp is at boiling point
> > - else if (temp === 212){
> > - //return "212 is at boiling point"
> > - return `${temp} is at boiling point`
> > - //Create else "broken"
> > - } else {
> > - return "broken"
> > - }
> > -}
> >
> > good job!
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419783050>
> > :
> >
> > >
> >
> > // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
> > // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
> >
> > -var arr = [3, 7, 0, 6, -9]
> > -//Create for loop
> > -for(let i=0; i<arr.length; i++){
> > - //console.log() and have the array being multiplied by 5
> > - console.log(arr[i] * 5)
> >
> > this one is *aaalllmost* there! yes, we are console logging the correct
> > values but a function requires a return. instead, you'd want to *push*
> > those values into a new array and then return that array. you're also going
> > t want to wrap all of this logic into a function:
> >
> > const multBy5 = (arr) => {
> > let multipliedValues = []
> > for(let i=0; i < arr.length; i++) {
> > multipliedValues.push(arr[i] * 5)
> > }
> > return multipliedValues
> > }console.log(multBy5(myNumbers1));
> >
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784425>
> > :
> >
> > >
> > // --------------------3) Create a function that multiplies each number in the array by 5 using map.
> > // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
> >
> > var myNumbers2 = [8, -7, 0, 6, 2]
> > -//create a const variable to make sure the variable name doesn't change
> > -//make it a function
> > -//use the map method to be able to go thorugh the array and multiply each index by 5
> > -const multiplyer = myNumbers2.map(value => value * 5)
> >
> > this one is also suuuuper close! this should just be wrapping a function.
> > right now it is just a variable with a const assignment. like this:
> >
> > const multiplyer = (arr) => arr.map(value => value * 5)
> >
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784594>
> > :
> >
> > > @@ -66,68 +35,26 @@ console.log(multiplyer);
> > var stringWithVowels1 = "HeyThereLearnStudent"
> > var stringWithVowels2 = "ILoveJavaScript"
> >
> > -//create const variable connected to the function and put the term string within the parenthesis
> > -const noVowels = (string) => {
> > - //create a let variable stating what vowels are
> > - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
> > - //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
> > - let array = string.split("")
> > - //create a return statement that will filter through the array and add in .join to join the constanants together
> > - return array.filter(value => vowels.indexOf(value) === -1).join("")
> > -}
> > -//console.log() both variables given
> > -console.log(noVowels(stringWithVowels1));
> > -console.log(noVowels(stringWithVowels2))
> >
> > good job!
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784925>
> > :
> >
> > >
> > -//create const variable connected to the function and put the term string within the parenthesis
> > -const noVowels = (string) => {
> > - //create a let variable stating what vowels are
> > - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
> > - //create a let variable that will filter through the array and split the indexes to be able to be easily accessed
> > - let array = string.split("")
> > - //create a return statement that will filter through the array and add in .join to join the constanants together
> > - return array.filter(value => vowels.indexOf(value) === -1).join("")
> >
> > this function needs a bit of refactoring to add some logic to check if the
> > input is actually a string
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785083>
> > :
> >
> > >
> > // --------------------6) Create a function that takes the toonimals array and returns an array with only the toon objects that are cats.
> > // Expected output: [ { name: "Stimpy", animal: "cat" }, { name: "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
> >
> > -//make all variables stackes above one another
> > -var toonimals = [
> > - {name: "Itchy", animal: "mouse"},
> > - {name: "Stimpy", animal: "cat"},
> > - {name: "Daffy", animal: "duck"},
> > - {name: "Scratchy", animal: "cat"},
> > - {name: "Ren", animal: "dog"},
> > - {name: "Felix", animal: "cat"}]
> > -//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find "cats"
> > -var animal = toonimals.filter(value => value.animal === "cat")
> > -//console.log() the code to call it
> > -console.log(animal);
> >
> > yay! your variable is checking for all the cats. now we just need to wrap
> > it all in a function to make it reusable.
> > ------------------------------
> >
> > In code-challenges.js
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785520>
> > :
> >
> > >
> > // --------------------7) Using the toonimals variable from #6, create a function that returns only the names of the non-cats.
> > // Expected output: "Itchy" "Daffy" "Ren"
> > -//make all variables stackes above one another
> > -var toonimals = [
> > - {name: "Itchy", animal: "mouse"},
> > - {name: "Stimpy", animal: "cat"},
> > - {name: "Daffy", animal: "duck"},
> > - {name: "Scratchy", animal: "cat"},
> > - {name: "Ren", animal: "dog"},
> > - {name: "Felix", animal: "cat"}]
> > -//create a variable function entitles animal - make it filter thorugh the indexes to only be able to find non cat names
> > -var animal = toonimals.filter(value => value.animal !== "cat")
> > -//console.log() the code to call it
> > -console.log(animal);
> >
> > almost! not quite but you're almost there. this solution again, is just a
> > variable, not a function. also, it will return the whole object of each
> > non-cat. we want just the *names* of the non-cats
> > ------------------------------
> >
> > In interview-questions.md
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785591>
> > :
> >
> > >
> > - Researched answer: A function is a set of instructions detailing how to do a task
> >
> > good research!
> > ------------------------------
> >
> > In interview-questions.md
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785777>
> > :
> >
> > >
> > - Researched answer: map function iterates through an array and returns a new array of the same length rather filter function loops through an array and returns a new array with only the values that are truthy or the values that satisfy your stated condition.
> >
> > good research! i hope this cleared it up a bit for you :)
> > ------------------------------
> >
> > In interview-questions.md
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785838>
> > :
> >
> > >
> >
> >
> > 3. What is the difference between console.log() and return?
> >
> > - Your answer: console.log() is used when debugging mostly because information gets inputed to the log that goes into the function to make the function do something, rather a return is part of the function.
> >
> > omg yes!
> > ------------------------------
> >
> > In interview-questions.md
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785953>
> > :
> >
> > >
> >
> >
> > 4. In regards to functions, what is an argument?
> >
> > - Your answer: an argument is what goes inside the () right?....
> >
> > it sure is! it is outside information that the function needs in order to
> > run
> > ------------------------------
> >
> > In interview-questions.md
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419786075>
> > :
> >
> > >
> > - Researched answer: Test-driven development is a software development process that relies on the repetition of a very short development cycle: requirements are turned into very specific test cases, then the code is improved so that the tests pass.
> >
> > yippee!
> >
> > —
> > You are receiving this because you authored the thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#pullrequestreview-405410218>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/APIFD7LQH42H6QOWELN6WHLRP5FNHANCNFSM4MY5BFZQ>
> > .
> >
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#issuecomment-623767933>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AMBXILFKOQG4XPOSE3BXDETRP5IRVANCNFSM4MY5BFZQ>.
>

jazmynCortez commented:
Ok sounds good

On Mon, May 4, 2020 at 4:56 PM Rachael Stanislaw <notifications@github.com>
wrote:

> Hmm you should be able to! We can walk through it tomorrow :)
>
> > On May 4, 2020, at 4:54 PM, jazmynCortez <notifications@github.com>
> wrote:
> >
> >
> > Oh no! How do I add that?
> >
> >
> > On Mon, May 4, 2020 at 4:27 PM Rachael Stanislaw <
> notifications@github.com>
> > wrote:
> >
> > > *@rachaelstanislaw* approved this pull request.
> > >
> > > yay! excellent work on your first assessment! i think we are missing
> the
> > > jest file though. let me know if you have questions about any of the
> > > feedback. then, you're good to merge your code to master :)
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419782696
> >
> > > :
> > >
> > > > - return `${temp} is below boiling point`
> > > - }
> > > - //Create an else if statement stating that temp is above boiling
> point
> > > - else if (temp > 212) {
> > > - //return "350 is above boiling point"
> > > - return `${temp} is above boiling point`
> > > - }
> > > - //Create an else if statement stating that temp is at boiling point
> > > - else if (temp === 212){
> > > - //return "212 is at boiling point"
> > > - return `${temp} is at boiling point`
> > > - //Create else "broken"
> > > - } else {
> > > - return "broken"
> > > - }
> > > -}
> > >
> > > good job!
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419783050
> >
> > > :
> > >
> > > >
> > >
> > > // --------------------2) Create a function that multiplies each
> number in the array by 5 using a for loop.
> > > // Use the test variable provided below. Expected outcome: [15, 35, 0,
> 30, -45]
> > >
> > > -var arr = [3, 7, 0, 6, -9]
> > > -//Create for loop
> > > -for(let i=0; i<arr.length; i++){
> > > - //console.log() and have the array being multiplied by 5
> > > - console.log(arr[i] * 5)
> > >
> > > this one is *aaalllmost* there! yes, we are console logging the correct
> > > values but a function requires a return. instead, you'd want to *push*
> > > those values into a new array and then return that array. you're also
> going
> > > t want to wrap all of this logic into a function:
> > >
> > > const multBy5 = (arr) => {
> > > let multipliedValues = []
> > > for(let i=0; i < arr.length; i++) {
> > > multipliedValues.push(arr[i] * 5)
> > > }
> > > return multipliedValues
> > > }console.log(multBy5(myNumbers1));
> > >
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784425
> >
> > > :
> > >
> > > >
> > > // --------------------3) Create a function that multiplies each
> number in the array by 5 using map.
> > > // Use the test variable provided below. Expected outcome: [40, -35,
> 0, 30, 10]
> > >
> > > var myNumbers2 = [8, -7, 0, 6, 2]
> > > -//create a const variable to make sure the variable name doesn't
> change
> > > -//make it a function
> > > -//use the map method to be able to go thorugh the array and multiply
> each index by 5
> > > -const multiplyer = myNumbers2.map(value => value * 5)
> > >
> > > this one is also suuuuper close! this should just be wrapping a
> function.
> > > right now it is just a variable with a const assignment. like this:
> > >
> > > const multiplyer = (arr) => arr.map(value => value * 5)
> > >
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784594
> >
> > > :
> > >
> > > > @@ -66,68 +35,26 @@ console.log(multiplyer);
> > > var stringWithVowels1 = "HeyThereLearnStudent"
> > > var stringWithVowels2 = "ILoveJavaScript"
> > >
> > > -//create const variable connected to the function and put the term
> string within the parenthesis
> > > -const noVowels = (string) => {
> > > - //create a let variable stating what vowels are
> > > - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
> > > - //create a let variable that will filter through the array and split
> the indexes to be able to be easily accessed
> > > - let array = string.split("")
> > > - //create a return statement that will filter through the array and
> add in .join to join the constanants together
> > > - return array.filter(value => vowels.indexOf(value) === -1).join("")
> > > -}
> > > -//console.log() both variables given
> > > -console.log(noVowels(stringWithVowels1));
> > > -console.log(noVowels(stringWithVowels2))
> > >
> > > good job!
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419784925
> >
> > > :
> > >
> > > >
> > > -//create const variable connected to the function and put the term
> string within the parenthesis
> > > -const noVowels = (string) => {
> > > - //create a let variable stating what vowels are
> > > - let vowels = ["a", "e", "i", "o", "u", "A", "E", "I", "O", "U"]
> > > - //create a let variable that will filter through the array and split
> the indexes to be able to be easily accessed
> > > - let array = string.split("")
> > > - //create a return statement that will filter through the array and
> add in .join to join the constanants together
> > > - return array.filter(value => vowels.indexOf(value) === -1).join("")
> > >
> > > this function needs a bit of refactoring to add some logic to check if
> the
> > > input is actually a string
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785083
> >
> > > :
> > >
> > > >
> > > // --------------------6) Create a function that takes the toonimals
> array and returns an array with only the toon objects that are cats.
> > > // Expected output: [ { name: "Stimpy", animal: "cat" }, { name:
> "Scratchy", animal: "cat" }, { name: "Felix", animal: "cat" } ]
> > >
> > > -//make all variables stackes above one another
> > > -var toonimals = [
> > > - {name: "Itchy", animal: "mouse"},
> > > - {name: "Stimpy", animal: "cat"},
> > > - {name: "Daffy", animal: "duck"},
> > > - {name: "Scratchy", animal: "cat"},
> > > - {name: "Ren", animal: "dog"},
> > > - {name: "Felix", animal: "cat"}]
> > > -//create a variable function entitles animal - make it filter thorugh
> the indexes to only be able to find "cats"
> > > -var animal = toonimals.filter(value => value.animal === "cat")
> > > -//console.log() the code to call it
> > > -console.log(animal);
> > >
> > > yay! your variable is checking for all the cats. now we just need to
> wrap
> > > it all in a function to make it reusable.
> > > ------------------------------
> > >
> > > In code-challenges.js
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785520
> >
> > > :
> > >
> > > >
> > > // --------------------7) Using the toonimals variable from #6, create
> a function that returns only the names of the non-cats.
> > > // Expected output: "Itchy" "Daffy" "Ren"
> > > -//make all variables stackes above one another
> > > -var toonimals = [
> > > - {name: "Itchy", animal: "mouse"},
> > > - {name: "Stimpy", animal: "cat"},
> > > - {name: "Daffy", animal: "duck"},
> > > - {name: "Scratchy", animal: "cat"},
> > > - {name: "Ren", animal: "dog"},
> > > - {name: "Felix", animal: "cat"}]
> > > -//create a variable function entitles animal - make it filter thorugh
> the indexes to only be able to find non cat names
> > > -var animal = toonimals.filter(value => value.animal !== "cat")
> > > -//console.log() the code to call it
> > > -console.log(animal);
> > >
> > > almost! not quite but you're almost there. this solution again, is
> just a
> > > variable, not a function. also, it will return the whole object of each
> > > non-cat. we want just the *names* of the non-cats
> > > ------------------------------
> > >
> > > In interview-questions.md
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785591
> >
> > > :
> > >
> > > >
> > > - Researched answer: A function is a set of instructions detailing how
> to do a task
> > >
> > > good research!
> > > ------------------------------
> > >
> > > In interview-questions.md
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785777
> >
> > > :
> > >
> > > >
> > > - Researched answer: map function iterates through an array and
> returns a new array of the same length rather filter function loops through
> an array and returns a new array with only the values that are truthy or
> the values that satisfy your stated condition.
> > >
> > > good research! i hope this cleared it up a bit for you :)
> > > ------------------------------
> > >
> > > In interview-questions.md
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785838
> >
> > > :
> > >
> > > >
> > >
> > >
> > > 3. What is the difference between console.log() and return?
> > >
> > > - Your answer: console.log() is used when debugging mostly because
> information gets inputed to the log that goes into the function to make the
> function do something, rather a return is part of the function.
> > >
> > > omg yes!
> > > ------------------------------
> > >
> > > In interview-questions.md
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419785953
> >
> > > :
> > >
> > > >
> > >
> > >
> > > 4. In regards to functions, what is an argument?
> > >
> > > - Your answer: an argument is what goes inside the () right?....
> > >
> > > it sure is! it is outside information that the function needs in order
> to
> > > run
> > > ------------------------------
> > >
> > > In interview-questions.md
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#discussion_r419786075
> >
> > > :
> > >
> > > >
> > > - Researched answer: Test-driven development is a software development
> process that relies on the repetition of a very short development cycle:
> requirements are turned into very specific test cases, then the code is
> improved so that the tests pass.
> > >
> > > yippee!
> > >
> > > —
> > > You are receiving this because you authored the thread.
> > > Reply to this email directly, view it on GitHub
> > > <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#pullrequestreview-405410218
> >,
> > > or unsubscribe
> > > <
> https://github.com/notifications/unsubscribe-auth/APIFD7LQH42H6QOWELN6WHLRP5FNHANCNFSM4MY5BFZQ
> >
> > > .
> > >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub <
> https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#issuecomment-623767933>,
> or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/AMBXILFKOQG4XPOSE3BXDETRP5IRVANCNFSM4MY5BFZQ
> >.
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/learn-academy-2020-bravo/week-1-assessment-jazmynCortez/pull/2#issuecomment-623768486>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APIFD7KPJTI7I6NDIL4K4JTRP5IZ7ANCNFSM4MY5BFZQ>
> .
>
